packages/experiments-realm/blog-post.gts

```diff
@@ -626,15 +641,12 @@ export class BlogPost extends CardDef {
             </p>
           {{/if}}
           <ul class='info'>
-            {{#if @model.authorBio}}
-              <li class='byline'>
-                <@fields.authorBio
-                  class='author'
-                  @format='atom'
-                  @displayContainer={{false}}
-                />
-              </li>
-            {{/if}}
+            <li class='byline'>
+              {{#each @fields.authors as |AuthorComponent index|}}
+                <AuthorComponent @format='atom' class='author' />
+                <span>{{this.additionalTextForAuthorComponent index}}</span>
+              {{/each}}
+            </li>
```

Comment:
I think this will need a `{{if @model.authors.length}}` wrapper around it, otherwise blank space shows up on the template when there are no authors